### PR TITLE
Master is now 9.0.0 development

### PR DESCRIPTION
**Pull request by nickvergessen**
Version number in master was lower then in stable8.2

@karlitschek @cmonteroluque @DeepDiver1975

### CODE REVIEW

File: version.php [hunk 1]
```diff
@@ -25,10 +25,10 @@
...
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
-$OC_Version = array(8, 2, 0, 9);
+$OC_Version = array(9, 0, 0, 0);
 
 // The human readable string
-$OC_VersionString = '8.2 RC1';
+$OC_VersionString = '9.0 pre alpha';
 
 $OC_VersionCanBeUpgradedFrom = array(8, 1);
```
Comment (PVince81) [inline version.php]: 8.2 here ?

**Comment by PVince81** [comment]
:+1:

**Comment by rullzer** [comment]
:+1:

**Comment by MorrisJobke** [comment]
:+1:

**Comment by PVince81** [comment]
@karlitschek @cmonteroluque please add this to the release checklist

**Comment by LukasReschke** [comment]
:+1:

**Comment by karlitschek** [comment]
:+1: yes

**Comment by cmonteroluque** [comment]
thank you @nickvergessen

**Pull request merged**